src/style.css

```diff
@@ -1,3 +1,60 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --link: #00acdc;
+}
+
 body {
   margin: 0;
+  background-color: rgb(216, 216, 216);
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: white;
+  width: 100%;
+}
+
+.container img {
+  margin: 10px 419px 0 20px;
```

Comment:
Avoid fixed margins as they can cause issues with responsiveness. Consider using relative units like 'em' or percentages, or applying margins to only one side consistently.